frontend/src/static/userProfileInfo.json

```diff
@@ -0,0 +1,11 @@
+[
+    { 
+        "idir" : "oliber",
+        "userInfo" :  
+            {
+                "name" : "Olga Liber",
+                "avatar" : "../assets/images/tyrannosaurus-rex1.png",
+                "email" : "olga.liberchuk@bc.gov.ca"
+            }
+    }
+]
```

Comment:
We are not hardcoding a user to display.
FAM is already working for user to login. This file should be deleted.